craft_providers/bases/ubuntu.py

```diff
@@ -97,7 +97,7 @@ def __init__(
 
         self._set_hostname(hostname)
 
-        self._packages = ["apt-utils", "curl", "fuse", "udev"]
+        self._packages = ["apt-utils", "build-essential", "curl", "fuse", "udev"]
```

Comment:
This is already installed on `buildd` images but installs extra packages on the `daily` interim images we are currently using?

Reply:
Yes, and apt will not try to install it again if it already installed.

Reply:
Great!